### PR TITLE
modprobe: Add page

### PR DESCRIPTION
**Pull request by sbrl**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/modprobe.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# modprobe
+
+> Add or remove modules from the Linux kernel.
+
+- Pretend to load a module into the Linux kernel, but don't actually do it:
```
Comment (agnivade) [inline pages/linux/modprobe.md]: Remove the word "LInux". I believe it is clear in the heading.

**Comment by dshuvar** [comment]
That interesting, why I saw `Oops! Command not found!` when I try to search modprobe and that's PR is was merged?

**Comment by pxgamer** [comment]
@dshuvar, what OS are you using?

**Comment by dshuvar** [comment]
> what OS are you using?

I using ubuntu and windows, with firefox browser, but what kind of impact can this have?

**Comment by pxgamer** [comment]
Oh, are you using a client? Or the website?

This page is in the `linux` directory, so if you're using the website you need to prefix it with `linux/` and if you're using Windows with a client, you'd need to use `--os linux` or `--linux`.

**Comment by dshuvar** [comment]
I have used the website, sure I found and check the markdown page in repo, but on the website not show me the page about that tool.

**Comment by pxgamer** [comment]
https://tldr.ostera.io/linux/modprobe
Commands that aren't in the `common` directory need to be searched for using a prefix. :slight_smile:

**Pull request merged**